src/data/zcl_abapgit_data_deserializer.clas.abap

```diff
@@ -73,6 +73,33 @@ CLASS zcl_abapgit_data_deserializer IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD is_table_allowed_to_edit.
+
+    DATA lv_tabname TYPE dd02l-tabname.
+
+    "Did the user flagged this table for update?
+    READ TABLE is_checks-overwrite TRANSPORTING NO FIELDS
+      WITH KEY obj_name = iv_table_name decision = zif_abapgit_definitions=>c_yes.
+    IF sy-subrc <>  0.
+      RETURN.
+    ENDIF.
```

Comment:
this check is related to the TABL object? ie. not the data, ie. the data cannot be updated unless the table is also changed?

Reply:
there's a separate checkbox for TABU but you are right that the read must include `obj_type = 'TABU'`